jobs/update-failed-distributions/statement_notification.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Notification from Business Registry</title>
+    <style>
+        /* Global Resets */
+        body, table, div, a, p {
+            font-family: Verdana, Arial, sans-serif;
+        }
+        body {
+            margin: 0;
+            padding: 0;
+        }
+        table {
+            border-collapse: collapse;
+        }
+        img {
+            border: none;
+        }
+    </style>
+    <meta name="referrer" content="origin-when-cross-origin"/>
+    <title>chat.pathfinder.gov.bc.ca</title>
```

Comment:
We have this in all of our templates (chat.pathfinder.gov.bc.ca). Can we start removing it ?

Reply:
hm.. will remove in this.. and create a techdebt for others

Reply:
done